golangci.yml

```diff
@@ -188,7 +181,26 @@ linters-settings:
       # - unlabelStmt
       # - unnamedResult
       # - wrapperFunc
-
+  gci:
+    # Section configuration to compare against.
+    # Section names are case-insensitive and may contain parameters in ().
+    # The default order of sections is `standard > default > custom > blank > dot`,
+    # If `custom-order` is `true`, it follows the order of `sections` option.
+    # Default: ["standard", "default"]
+    sections:
+      - standard # Standard section: captures all standard packages.
+      - default # Default section: contains all imports that could not be matched to another section type.
+      - prefix(open-cluster-management.io) # Custom section: groups all imports with the specified Prefix.
```

Comment:
The upstream ocm has a very strict package import order check:
1. standard packages
2. 3rd party packages
3. ocm org packages
4. ocm local repo packages

Examples: https://github.com/open-cluster-management-io/ocm/blob/main/pkg/operator/operators/klusterlet/controllers/klusterletcontroller/klusterlet_cleanup_controller.go#L4-L31

I don't know if this kind of strict rule is welcomed for the import controller repo.

Reply:
That's a good point, I didn't add gci this time.